content/cloud/changelog/cloud-custom-domains.md

```diff
@@ -0,0 +1,20 @@
+title = "Fermyon Cloud Custom Domains"
+template = "changelog_item"
+date = "2023-07-26T16:00:00Z"
+enable_shortcodes = true
+tags = ["networking"]
+[extra]
+type= "changelog_post"
+
+---
+
+Blog authors and website tinkerers can rest easy knowing that Fermyon Cloud now supports custom domains. Until today, Spin applications were limited to either default randomly generated domain names or custom Fermyon subdomains; however, we heard your feedback that custom domains were an important feature to bring blogs and personalized websites to Fermyon Cloud. Using a custom domain name with your Spin application can improve your workload's SEO, build brand integrity, and improve discoverability. We encourage developers to bring their custom domains to Fermyon Cloud and assign them to their favorite Spin applications. Slat's has already gotten a headstart at [finickywhiskers.com](https://www.finickywhiskers.com/index.html). Let us know your thoughts on the feature at [feedback@fermyon.com](mailto:feedback@fermyon.com). 
```

Comment:
At the time I wrote this, I thought it was funny+light hearted but now I worry it might come across a bit condescending. 
```suggestion
Fermyon Cloud is excited to share that it now supports custom domains. Until today, Spin applications were limited to either default randomly generated domain names or custom Fermyon subdomains; however, we heard your feedback that custom domains were an important feature to bring blogs and personalized websites to Fermyon Cloud. Using a custom domain name with your Spin application can improve your workload's SEO, build brand integrity, and improve discoverability. We encourage developers to bring their custom domains to Fermyon Cloud and assign them to their favorite Spin applications. Slat's has already gotten a headstart at [finickywhiskers.com](https://www.finickywhiskers.com/index.html). Let us know your thoughts on the feature at [feedback@fermyon.com](mailto:feedback@fermyon.com). 
```